src/platform/plugins/shared/discover/public/application/main/state_management/redux/actions/save_discover_session.test.ts

```diff
@@ -56,8 +56,10 @@ const setup = ({
     );
   const dataViewCreateSpy = jest.spyOn(services.dataViews, 'create');
   const dataViewsClearCacheSpy = jest.spyOn(services.dataViews, 'clearInstanceCache');
+  const savedSearch = { ...savedSearchMock, timeRestore: false };
+  savedSearch.searchSource.setField('filter', []);
```

Comment:
This line was added because calling `updateSavedSearch` in `getDiscoverStateMock` causes `filter` to default to `[]`, which breaks tests in this file that use `savedSearch` for comparisons.

Won't be an issue when these tests are migrated to use `getDiscoverInternalStateMock` which properly initializes state in the same way the app does.